packages/gtk3.rb

```diff
@@ -49,26 +51,33 @@ def self.build
             builddir"
     system 'meson configure builddir'
     system 'ninja -C builddir'
-    @file = <<~GTK3_CONFIG_HEREDOC
-      [Settings]
-      gtk-application-prefer-dark-theme = false
-      gtk-icon-theme-name = hicolor
-      gtk-fallback-icon-theme = gnome
-      gtk-font-name = Arial 10
-    GTK3_CONFIG_HEREDOC
   end
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
     system "sed -i 's,null,,g'  #{CREW_DEST_LIB_PREFIX}/pkgconfig/gtk*.pc"
     FileUtils.mkdir_p("#{CREW_DEST_HOME}/.config/gtk-3.0")
-    File.write("#{CREW_DEST_HOME}/.config/gtk-3.0/settings.ini", @file)
+    File.write("#{CREW_DEST_HOME}/.config/gtk-3.0/settings.ini", @gtk3settings)
   end
 
   def self.postinstall
     # generate schemas
     system "glib-compile-schemas #{CREW_PREFIX}/share/glib-2.0/schemas"
     # update mime database
     system "update-mime-database #{CREW_PREFIX}/share/mime"
+    @xdg_config_home = ENV['XDG_CONFIG_HOME']
+    @gtk3settings = <<~GTK3_CONFIG_HEREDOC
+      [Settings]
+      gtk-application-prefer-dark-theme = false
+      gtk-icon-theme-name = hicolor
+      gtk-fallback-icon-theme = gnome
+      gtk-font-name = Arial 10
+    GTK3_CONFIG_HEREDOC
+    unless File.exist?("#{@xdg_config_home}/gtk-3.0/settings.ini")
+      puts
+      puts 'Adding basic gtk4 settings to XDG_CONFIG_HOME/gtk-3.0/settings.ini'.lightblue
+      FileUtils.mkdir_p "#{@xdg_config_home}/gtk-3.0"
+      File.write("#{@xdg_config_home}/gtk-3.0/settings.ini", @gtk3settings)
```

Comment:
Didn't we already install this config?  In any event, it probably should be included in the package files anyway.  Maybe add a `def self.remove` to optionally remove the config?

Reply:
You don't want to overwrite local customizations, which are likely to happen with gtk. Simple things like changing the default font would get overwritten.

I would hesitate on a self-remove for this. We're not talking about a large install like you have with wine in the config directory.  The config file is measured in bytes...

Reply:
Actually, why aren't we installing the system-wide versions of these files?

Reply:
Stand by...

Reply:
ok should be good now.